app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -170,16 +206,39 @@ def with_disk_attachments_service(vm)
   end
 
   class_methods do
+    def api3_supported_features
+      []
+    end
+
+    def api4_supported_features
+      [:reconfigure_vms, :snapshots]
```

Comment:
so, will you rename `reconfigure_vms` on this PR or on a follow up one?
because I rather dont want to let features in that sound not right.
and it looks like the features come directly from the `probe` method.
maybe you need some mapping later anyway.

how about removing this for now?